Presentation.Web/Tests/Helpers/SideNavigation/LocalItSystemNavigation.ts

```diff
@@ -12,7 +12,9 @@
         return element(by.css(`[data-ui-sref="${LocalItSystemNavigationSrefs.relationsSref}"`)).click();
     }
     
-    
+    public static openArchivingPage() {
+        return element(by.css(`[data-ui-sref="${LocalItSystemNavigationSrefs.archivingSref}"`)).click();
```

Comment:
Kunne du lave noget lign. 
private static  getSubMenuElement(srefName: string) {
        return element(by.css(`[data-ui-sref="${srefName}"`));
    }
som der er gjort i ItSystemNavigation.ts filen? Måske samle den i en klasse for sig selv som de forskellige subnavigation filer kan importe?

Reply:
tjo det gør nok livet nemmere

Reply:
fixed